src/teaching-element/Assessment/TextResponse.vue

```diff
@@ -25,6 +28,11 @@ export default {
       userAnswer: this.submission
     };
   },
+  computed: {
+    uniqueId() {
+      return `text-response-${random(0, 10000)}`;
```

Comment:
Why random instead of lodash uniqueId? 🤔

Reply:
I didn't read the docs carefully.